docs/TESTS.md

```diff
@@ -1,15 +1,11 @@
 # Tests
 
-<!-- TODO: write document
+## Running tests
+To run the tests run the command
 
-  This document should describe everything related to running tests in the track.
+```bash
+./bin/test
+```
```

Comment:
The intended audience for this file is a student, and it should describe how the student can run an exercise's tests. 
Looking at an exercise, I don't see a `bin/test` file in it.
However, this repo _does_ have a `bin/test` file, so you might have written this describing how to test the track?
If so, could you rewrite this to describe how the student should run the tests?

Reply:
Ah yes I did describe how to test the track rather than individual exercises. Will update, thanks!

Reply:
Fixed in 88476d02dc1b5ddde5161265cc293f279bbf7e79